### ISSUE TITLE
[V 15.7.179.6572] Log.LogMessage/Warning/Error does not send the log message to main MSBuild instance when running in paralel.

### ISSUE THREAD

**Issue by AraHaan** [Microsoft/msbuild#3409]
### Steps to reproduce

Either include a project sample, attach a zipped project, or provide IDE / CLI steps to create the project and repro the behaviour. Project files are at https://github.com/Elskom/Els_kom_new/blob/master/PCbuild/

Directory contents:
See the directory structure of https://github.com/Elskom/Els_kom_new/

Command line
```
msbuild PCbuild/pcbuild.sln /p:Configuration=Release /nologo /verbosity:m /m
```
### Expected  behavior
For the logging functions to output to the main MSBuild instance showing the task status on each project (with the project file name appended to end like so ' - <project name>')

### Actual behavior
No task logs are outputted in the main instance, they go to the paralel instances which are never read at all either.

### Environment data
`msbuild /version` output:
Microsoft (R) Build Engine version 15.7.179.6572 for .NET Framewor
Copyright (C) Microsoft Corporation. All rights reserved.

15.7.179.6572

OS info:
Windows 7 Ultimate SP1 Build 7601

If applicable, version of the tool that invokes MSBuild (Visual Studio, dotnet CLI, etc):
command line and Visual Studio 2017 (I had to manually add MSBuild from VS2017 to path).

**Comment by rainersigwald** [comment]
I don't see any meaningful difference here. What messages seem to be missing?

```
C:\Users\raines\source\repos\Els_kom_new>msbuild PCbuild/pcbuild.sln /p:Configuration=Release /nologo /verbosity:m /m
  Els_kom_Core -> C:\Users\raines\source\repos\Els_kom_new\bin\x86\Release\Els_kom.Core.dll
  Running Code Analysis...
  Code Analysis Complete -- 0 error(s), 0 warning(s)
  komv4_plugin -> C:\Users\raines\source\repos\Els_kom_new\bin\x86\Release\plugins\komv4.plugin.dll
  komv3_plugin -> C:\Users\raines\source\repos\Els_kom_new\bin\x86\Release\plugins\komv3.plugin.dll
  komv2_plugin -> C:\Users\raines\source\repos\Els_kom_new\bin\x86\Release\plugins\komv2.plugin.dll
  Running Code Analysis...
  Running Code Analysis...
  Running Code Analysis...
  Code Analysis Complete -- 0 error(s), 0 warning(s)
  Code Analysis Complete -- 0 error(s), 0 warning(s)
  Code Analysis Complete -- 0 error(s), 0 warning(s)
  callbacktest_plugin -> C:\Users\raines\source\repos\Els_kom_new\bin\x86\Release\plugins\callbacktest.plugin.dll
  Running Code Analysis...
  Code Analysis Complete -- 0 error(s), 0 warning(s)
  Els_kom -> C:\Users\raines\source\repos\Els_kom_new\bin\x86\Release\Els_kom.exe
  Running Code Analysis...
  Code Analysis Complete -- 0 error(s), 0 warning(s)
  packbuild -> C:\Users\raines\source\repos\Els_kom_new\bin\x86\Release\packbuild.exe
  Running Code Analysis...
  Code Analysis Complete -- 0 error(s), 0 warning(s)
  Writing build files to Els_kom_new.zip.
  Writing debug symbol files to Symbols.zip.

C:\Users\raines\source\repos\Els_kom_new>
C:\Users\raines\source\repos\Els_kom_new>msbuild PCbuild/pcbuild.sln /p:Configuration=Release /nologo /verbosity:m
  Els_kom_Core -> C:\Users\raines\source\repos\Els_kom_new\bin\x86\Release\Els_kom.Core.dll
  Running Code Analysis...
  Code Analysis Complete -- 0 error(s), 0 warning(s)
  Els_kom -> C:\Users\raines\source\repos\Els_kom_new\bin\x86\Release\Els_kom.exe
  Running Code Analysis...
  Code Analysis Complete -- 0 error(s), 0 warning(s)
  komv2_plugin -> C:\Users\raines\source\repos\Els_kom_new\bin\x86\Release\plugins\komv2.plugin.dll
  Running Code Analysis...
  Code Analysis Complete -- 0 error(s), 0 warning(s)
  komv3_plugin -> C:\Users\raines\source\repos\Els_kom_new\bin\x86\Release\plugins\komv3.plugin.dll
  Running Code Analysis...
  Code Analysis Complete -- 0 error(s), 0 warning(s)
  komv4_plugin -> C:\Users\raines\source\repos\Els_kom_new\bin\x86\Release\plugins\komv4.plugin.dll
  Running Code Analysis...
  Code Analysis Complete -- 0 error(s), 0 warning(s)
  callbacktest_plugin -> C:\Users\raines\source\repos\Els_kom_new\bin\x86\Release\plugins\callbacktest.plugin.dll
  Running Code Analysis...
  Code Analysis Complete -- 0 error(s), 0 warning(s)
  packbuild -> C:\Users\raines\source\repos\Els_kom_new\bin\x86\Release\packbuild.exe
  Writing build files to Els_kom_new.zip.
  Writing debug symbol files to Symbols.zip.
```

**Comment by AraHaan** [comment]
These messages:
https://github.com/Elskom/Els_kom_new/blob/master/Els_kom_Core/Els_kom_Core.csproj#L147
https://github.com/Elskom/Els_kom_new/blob/master/Els_kom_Core/Els_kom_Core.csproj#L243
https://github.com/Elskom/Els_kom_new/blob/master/Els_kom_Core/Els_kom_Core.csproj#L335

Also realized another issue but that is in that the rc.exe finder task. Like an idiot my compile failure when not running in parallel to Els_kom.exe is due to not doing proc.WaitForExit in the task making it fail due to rc.exe having the resource file still opened. But at least I now know about that issue to fix it. But the parallel build issue still remains unless I switch from parallel builds for now to get those logs to show up like I want.

**Comment by rainersigwald** [comment]
I see:

```
C:\Els_kom_new>msbuild PCbuild/pcbuild.sln /p:Configuration=Release /nologo /verbosity:m /m
  Getting build info from git
  Dumping Resources Path for Forms Designer.
CSC : error CS2001: Source file 'C:\Els_kom_new\Els_kom_Core\..\externals\ZLIB.NET\Inflate.cs' could not be found. [C:\Els_kom_new\Els_kom_Core\Els_kom_Core.csproj]
CSC : error CS2001: Source file 'C:\Els_kom_new\Els_kom_Core\..\externals\ZLIB.NET\InfTree.cs' could not be found. [C:\Els_kom_new\Els_kom_Core\Els_kom_Core.csproj]
CSC : error CS2001: Source file 'C:\Els_kom_new\Els_kom_Core\..\externals\ZLIB.NET\StaticTree.cs' could not be found. [C:\Els_kom_new\Els_kom_Core\Els_kom_Core.csproj]
CSC : error CS2001: Source file 'C:\Els_kom_new\Els_kom_Core\..\externals\ZLIB.NET\SupportClass.cs' could not be found. [C:\Els_kom_new\Els_kom_Core\Els_kom_Core.csproj]
CSC : error CS2001: Source file 'C:\Els_kom_new\Els_kom_Core\..\externals\ZLIB.NET\Tree.cs' could not be found. [C:\Els_kom_new\Els_kom_Core\Els_kom_Core.csproj]
CSC : error CS2001: Source file 'C:\Els_kom_new\Els_kom_Core\..\externals\ZLIB.NET\ZInputStream.cs' could not be found. [C:\Els_kom_new\Els_kom_Core\Els_kom_Core.csproj]
CSC : error CS2001: Source file 'C:\Els_kom_new\Els_kom_Core\..\externals\ZLIB.NET\Zlib.cs' could not be found. [C:\Els_kom_new\Els_kom_Core\Els_kom_Core.csproj]
CSC : error CS2001: Source file 'C:\Els_kom_new\Els_kom_Core\..\externals\ZLIB.NET\ZOutputStream.cs' could not be found. [C:\Els_kom_new\Els_kom_Core\Els_kom_Core.csproj]
CSC : error CS2001: Source file 'C:\Els_kom_new\Els_kom_Core\..\externals\ZLIB.NET\ZStream.cs' could not be found. [C:\Els_kom_new\Els_kom_Core\Els_kom_Core.csproj]
CSC : error CS2001: Source file 'C:\Els_kom_new\Els_kom_Core\..\externals\ZLIB.NET\ZStreamException.cs' could not be found. [C:\Els_kom_new\Els_kom_Core\Els_kom_Core.csproj]
CSC : error CS2001: Source file 'C:\Els_kom_new\Els_kom_Core\..\externals\ZLIB.NET\InfCodes.cs' could not be found. [C:\Els_kom_new\Els_kom_Core\Els_kom_Core.csproj]
CSC : error CS2001: Source file 'C:\Els_kom_new\Els_kom_Core\..\externals\ZLIB.NET\Adler32.cs' could not be found. [C:\Els_kom_new\Els_kom_Core\Els_kom_Core.csproj]
CSC : error CS2001: Source file 'C:\Els_kom_new\Els_kom_Core\..\externals\ZLIB.NET\Deflate.cs' could not be found. [C:\Els_kom_new\Els_kom_Core\Els_kom_Core.csproj]
CSC : error CS2001: Source file 'C:\Els_kom_new\Els_kom_Core\..\externals\ZLIB.NET\InfBlocks.cs' could not be found. [C:\Els_kom_new\Els_kom_Core\Els_kom_Core.csproj]
```

Was the problem fixed by https://github.com/Elskom/Els_kom_new/pull/81/commits/efa5b7adc170dc01f6e5382aead9a3d2ed6c517d in https://github.com/Elskom/Els_kom_new/pull/81? You were passing `/verbosity:m[inimal]`, which doesn't log message importances below `High`.

**Comment by CatGirlsAreLife** [comment]
I believe so yes, I did not realize I forgot the MessageImportance param.
